Makefile

```diff
@@ -84,6 +85,14 @@ start-seed-admission-controller:
 start-gardenlet:
 	@./hack/local-development/start-gardenlet
 
+.PHONY: start-landscaper-gardenlet-reconcile
+start-landscaper-gardenlet-reconcile:
+	@./hack/local-development/start-landscaper-gardenlet RECONCILE
+
+.PHONY: start-landscaper-gardenlet-delete
+start-landscaper-gardenlet-delete:
+	@./hack/local-development/start-landscaper-gardenlet DELETE
+
```

Comment:
What about using a `LANDSCAPER_OPERATION` variable and only have one `start-landscaper-gardenlet` rule?